### DISCUSSION TITLE
Hide Recommended section in dev builds too

### DISCUSSION THREAD

**Discussion by Yoad1704** [valinet/ExplorerPatcher#995] [Q&A]
Hi, I've been using explorer patcher for a while (which I think is great), and now the pre-release because I was excited to try the new features (such as hiding the recommended section), in a regular build of windows but now, I entered the dev channel to experience some of the upcoming features that are in windows (such as folder previews). Unfortunately, the "hide recommended section" doesn't work **for me** (I don't know about other people) in the dev build... Is there a way I can make it work in the dev build or is it just doesn't support that at the moment?

**Comment by valinet** [comment]
Yeah, it wasn’t tested on Insider Preview, probably the Pinned and Recommend sections’ locations in the UI tree are different.

**Reply by Yoad1704** [reply]
Will there be a fix for that?

**Reply by valinet** [reply]
At some point, there will be.

**Reply by Yoad1704** [reply]
Do you know when? Soon or not?

**Reply by valinet** [reply]
Is it really necessary to be pushy?

**Reply by Yoad1704** [reply]
No, sorry - I just need this feature here considering that I can't leave it by now....

**Reply by valinet** [reply]
I have fixed this in the latest pre-release (42.30).

**Reply by Yoad1704** [reply]
Thank you so much! sorry for being pushy....

**Reply by Yoad1704** [reply]
Hi again, wanted to inform you that in the latest dev build (22572) neither the "hide recommended section" nor the windows 10 start menu work now. Of course, I didn't test all of the features since some aren't compatible with another program I use so I don't know about anything else. I think it will be better to check it out and find a solution like last time. Thank you

**Reply by valinet** [reply]
I checked it myself yesterday, it was working. My advice is, as usual, to first test on a clean, untouched VM, in order to exclude variables.

**Reply by Yoad1704** [reply]
Are you sure you checked it on the latest build (22572)? I installed it as an update and the explorer patcher was already included for me there. Maybe it's because I didn't update it?

**Reply by Yoad1704** [reply]
Never mind, checked again, and it's working. Sorry for the false report

**Reply by lfloesser** [reply]
Hey @valinet, i have the same issue on the newest Insider build  23606.1000 , the hide recommended section part is not working, I'd love to have a quick fix!
Best Wishes
